src/gcp/cloudfunctions.js

```diff
@@ -1,7 +1,7 @@
 "use strict";
 
 var api = require("../api");
-
+const FirebaseError = require("../error");
```

Comment:
If you have a moment, make the rest of these `const` as well, and sort them :)

Reply:
Done.